### PR TITLE
civichq-api-81: App Technology field

### PR DESCRIPTION
**Pull request by paul-ghiran**
Changed `/addapp` `/approfile` routes and the Apps db schema to to allow the usage of the "Technologies" field as seen in #81

### CODE REVIEW

File: api/server.js [hunk 1]
```diff
@@ -26,7 +26,8 @@ var port = appConfig.get('port');        // set our port
 var routesToAuthorize = ['/api/updateapp', '/api/appprofile', 
                         '/api/uploadlogo', '/api/addapp', 
                         '/api/masterprofile', '/api/categories',
-                        '/api/approvedapps', '/api/tags', '/api/search'];
+                        '/api/approvedapps', '/api/tags',
+                        '/api/search', '/api/technologies'];
```
Comment (lrdTnc) [inline api/server.js]: no router and service defined for path 'api/technologies'. is this truly needed?

Comment (lrdTnc) [reply api/server.js]: never mind.  I saw this afterwards: https://github.com/code4romania/civichq-api/pull/87

**Comment by aniri** [comment]
@paul-ghiran Could you please check the merge conflicts?

Also, the missing editapp call was finally merged into the develop branch! It would be great if you could update that call as well. If not, let me know and I will add a separate issue for that.

Thanks!!

**Comment by RaduCStefanescu** [comment]
@paul-ghiran can you please check the merge conflicts so we can merge this PR? Thanks!

**Comment by aniri** [comment]
@RaduCStefanescu @paul-ghiran fixed merge conflicts and merging the pull request! wooh! thanks!

**Comment by paul-ghiran** [comment]
I missed the notifications for this one, sorry :( Thanks @aniri !

**Pull request merged**